armi/reactor/parameters/parameterDefinitions.py

```diff
@@ -26,16 +26,15 @@
 --------
 armi.reactor.parameters
 """
+from typing import Any, Dict, Optional, Sequence, Tuple, Type
```

Comment:
"from x" comes before "import x" alphabetically.  That's all.

So "from" imports are usually placed before normal "import"s.

To be clear, I don't really have a dog in this fight.  But ARMI has (slowly) be adjusting all it's imports to follow the same rules, and these are the rules that are standard.  I have no strong opinions about which would have been a better standard.

(I think this is the standard because most IDEs allow you to just highlight a block of text and hit and "alphabetize" button. Which is what I did with this block of imports here. Probably, it doesn't go deeper than that: convenience.)

Reply:
> (I think this is the standard because most IDEs allow you to just highlight a block of text and hit and "alphabetize" button. Which is what I did with this block of imports here.

🤯 how did I not know this?? My life with imports have forever changed 😄

Reply:
![image](https://github.com/terrapower/armi/assets/21959600/abf6e876-9c44-4e59-afcf-b86b90039423)

Reply:
@keckler are you suggesting this be changed in some way?

Reply:
I'm saying that our developer docs provide an example that is counter to what is being suggested here. 

I was always under the impression that `from` imports went after plain module imports, and I probably got that impression from the developer docs that I screenshotted above.

Reply:
I do not mind either way, but similar to Tony, I was previously under a different impression.

Change away, not important!